src/layerelement.cpp

```diff
@@ -1622,6 +1612,34 @@ int LayerElement::LayerCountInTimeSpan(FunctorParams *functorParams)
     return (this->Is(CHORD)) ? FUNCTOR_SIBLINGS : FUNCTOR_CONTINUE;
 }
 
+int LayerElement::LayerElementsInTimeSpan(FunctorParams *functorParams)
+{
+    LayerElementsInTimeSpanParams *params = static_cast<LayerElementsInTimeSpanParams *>(functorParams);
+    assert(params);
+
+    Layer *currentLayer = static_cast<Layer *>(GetFirstAncestor(LAYER));
+    if (!currentLayer || (currentLayer != params->m_layer) || IsScoreDefElement() || Is(MREST)) 
+        return FUNCTOR_SIBLINGS;
+    if (!GetDurationInterface() || Is(MSPACE) || Is(SPACE) || HasSameasLink())
+        return FUNCTOR_CONTINUE;
+
+    const double duration = !GetParent()->Is(CHORD)
+        ? GetAlignmentDuration(params->m_mensur, params->m_meterSig)
+        : static_cast<Chord *>(GetParent())->GetAlignmentDuration(params->m_mensur, params->m_meterSig);
```

Comment:
This will need to be a `vrv_cast` once merged with the latest version of develop. Not sure what is the best way to resolve it.